string topic
builtin_interfaces/Time start
uint64 num_messages
uint64 num_bytes

float64 window_duration_secs
float64 window_bandwidth
float64 window_rate
float64 window_average_delta

